javascript課題①-3.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+
+    const tasks = ["掃除","買い物","散歩"];
+
+    function tasklist() {
+    console.log("=========================");
+    console.log("現在持っているタスク一覧");
+    console.log("=========================");
+    tasks.forEach(function(item, index, array) {
```

Comment:
ここもアロー関数を利用した方がいいと思います。
あと、第3引数は利用していない様なので、削除してください。

```suggestion
    tasks.forEach((item, index) => {
```